### PR TITLE
Fix gulp-util dependency issue (work in progress)

### PR DESCRIPTION
**Pull request by rinella**

### CODE REVIEW

**Comment by shaneog** [comment]
@kevva `gulp-util` is deprecated - see official post here: https://medium.com/gulpjs/gulp-util-ca3b1f9f9ac5

Would it be possible to merge this PR and release a new version?

**Comment by rinella** [comment]
I did not remove the download dependency yet.  The build was broken so I fixed that as a first step.

File: index.js [hunk 1]
```diff
@@ -40,15 +41,15 @@ module.exports = BinWrapper;
  */
 
 BinWrapper.prototype.src = function (src, os, arch) {
-	if (!arguments.length) {
+	if (!arguments.length === true) {
```
Comment (sindresorhus) [inline index.js]: This is not correct

**Comment by rinella** [comment]
i believe the code is working (i am able to download files), but a few of the tests return : "Promise returned by test never resolved".  i'm not sure why.  

/cc @sindresorhus

**Comment by rejas** [comment]
How is this issue going on? Anyone still on it? Help needed?

**Comment by rejas** [comment]
hi @rinella [Downloadv7.1.0](https://github.com/kevva/download/releases/tag/v7.1.0) got released, maybe you can update your PR accordingly and rebase it agaisnt the current master?

**Comment by rinella** [comment]
not having any luck with the tests using Download v7.  closing PR due to lack of time.

**Comment by rejas** [comment]
Thx anyway.

**Pull request closed without merging**